customer-bill/src/main/java/org/fiware/tmforum/customer_bill/domain/RelatedPartyRef.java

```diff
@@ -0,0 +1,41 @@
+package org.fiware.tmforum.customer_bill.domain;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+import org.fiware.tmforum.customer_bill.domain.customer_bill.AppliedCustomerBillingRate;
+import org.fiware.tmforum.customer_bill.domain.customer_bill.CustomerBill;
+import org.fiware.tmforum.customer_bill.domain.customer_bill.CustomerBillOnDemand;
+import org.fiware.tmforum.mapping.annotations.*;
+
+import java.util.List;
+
```

Comment:
RelatedParty can be a customer bill? Sounds weird, wouldn't make Individual or Organisation or customer or what ever more sense?
Since its only a Ref to  a RelatedParty(e.g. does not need to be materialized into a concrete object), it might not neccessarily need to be a RefEntity.